examples/Transfer_Learning/basic_tl.py

```diff
@@ -0,0 +1,176 @@
+## Transfer Learning
+
+# This example demonstrates BayBE's
+# {doc}`Transfer Learning </userguide/transfer_learning>` capabilities:
+# * We construct a campaign,
+# * give it access to data from a related but different task,
+# * and show how this additional information boosts performance.
+
+### Imports
+
+import json
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from botorch.test_functions.synthetic import Hartmann
+
+from baybe import Campaign
+from baybe.objective import Objective
+from baybe.parameters import NumericalDiscreteParameter, TaskParameter
+from baybe.searchspace import SearchSpace
+from baybe.simulation import simulate_scenarios
+from baybe.targets import NumericalTarget
+from baybe.utils.botorch_wrapper import botorch_function_wrapper
+
+### Settings
+
+# The following settings are used to set up the problem:
```

Comment:
Do we really need all of the comments in the following lines? Most of the names are pretty self-explanatory. I'd honestly prefer an explanation of `SMOKE_TEST`, since this is something that might not be clear (at least until we have that in all examples)

Reply:
Discussed in the meeting. There "comments" were already there in your previous text, my changes only structured them into inline comments. An no explanation needed for `SMOKE_TEST` imo since this is super common in example files.